From 352a53fa4f89f8c330d45bc7c4180026c2a51b1a Mon Sep 17 00:00:00 2001
From: Zhang Jiejing <jiejing.zhang@freescale.com>
Date: Wed, 20 Apr 2011 10:47:42 +0800
Subject: [PATCH 897/998] ENGR00142347-2 MX53_SMD: add EETI eGalax touch screen support.

Add EETI eGalax touch screen support for MX53_SMD board.

Signed-off-by: Zhang Jiejing <jiejing.zhang@freescale.com>
---
 arch/arm/mach-mx5/mx53_smd.c |    5 +++++
 1 files changed, 5 insertions(+), 0 deletions(-)

diff --git a/arch/arm/mach-mx5/mx53_smd.c b/arch/arm/mach-mx5/mx53_smd.c
index 5e0d18b..0d08abe 100644
--- a/arch/arm/mach-mx5/mx53_smd.c
+++ b/arch/arm/mach-mx5/mx53_smd.c
@@ -728,6 +728,11 @@ static struct i2c_board_info mxc_i2c2_board_info[] __initdata = {
 	.platform_data = &p1003_ts_data,
 	},
 	{
+		.type = "egalax_ts",
+		.addr = 0x4,
+		.irq = gpio_to_irq(MX53_SMD_CAP_TCH_INT1),
+	},
+	{
 	.type = "isl29023",
 	.addr = 0x44,
 	.irq  = gpio_to_irq(MX53_SMD_ALS_INT),
-- 
1.7.4.1

